server/src/rooms/data/roomData.json

```diff
@@ -150,15 +150,15 @@
+    "description": "You enter a seedy back alleyway where shifty-eyed people in navy blue trench coats all try to swindle those foolish enough to participate in their visibly-rigged games of chance. Several of the people running these tables are very obviously removing cerulean balls from under cups and pulling extra cards from their sleeves. Every time someone does this, the crowd claps in glee, clearly amused by what's happening. You're not sure you understand, but they seem to be enjoying it. <br/><br/>Fortunately, you see a bright blue table of legitimate games that you do understand. These are at a table labelled 'Steam Sale', and you can peruse them at your leisure.<a href=\"https://store.steampowered.com/curator/41222044/sale/RoguelikeCelebration2025 \" target=\"_blank\">Visit the Steam Sale!</a><br/><br/>On an empty side table nearby rests a Knight.<br/><br/>You can also return to the [[Carnival Games]].",
 	"hidden": false,
     "roomId": "gameTables"
   },   
   "merryGoRound": {
     "displayName": "Merry-Go-Round",
     "shortName": "merry go round",
     "id": "merryGoRound",
-    "description": "You approach a massive, spinning machine styled after a merry-go-round, but where you might expect there to be large poles attached to floor and ceiling suspending a statue of an animal, you instead just see free-roaming beasts and an assortments of seats. Those seats are designed not for human proportions, but for these beasts. Indeed, you see that each beast appears to draw the eye of a potential rider, and then brings them dutifully to its chosen nest. The beast then waits to be sat upon by the rider. The ride has no one managing it, the creatures seem to be in charge of themselVes, and are quite insistent on this riding-based relationship.<br/><br/>You can also return to the [[Carnival Games]].",
+    "description": "You approach a massive, spinning machine styled after a merry-go-round. Where you might expect there to be large poles attached to the floor and ceiling suspending a statue of an animal, you instead just see free-roaming beasts and an assortment of bright green seats. Those seats are designed not for human proportions, but for these beasts. Indeed, you see that each beast appears to draw the eye of a potential rider, and then brings them dutifully to its chosen nest. The beast then waits to be sat upon by the rider, an emerald saddle on its back. The ride has no one managing it, the creatures seem to be in charge of themselVes, and are quite insistent on this riding-based relationship.<br/><br/>In the place of one of the seats, you instead find an oversized pawn.<br/><br/>You can also return to the [[Carnival Games]].",
```

Comment:
Should "pawn" be capitalized here? All the other chess piece names are capitalized, looks like.

<img width="714" height="69" alt="image" src="https://github.com/user-attachments/assets/c574583e-6262-4e17-8698-1d838d77d489" />